src/onnx/parse_slice.cpp

```diff
@@ -30,7 +32,14 @@ struct parse_slice : op_parser<parse_slice>
             step_arg.visit([&](auto s) { steps.assign(s.begin(), s.end()); });
             if(!std::all_of(steps.begin(), steps.end(), [](auto s) { return s == 1; }))
             {
-                MIGRAPHX_THROW("PARSE_SLICE: cannot handle step other than 1");
+                if(std::all_of(steps.begin(), steps.end(), [](auto s) { return s == -1; }))
```

Comment:
This should check `abs(s) == 1` then throw the error.

Reply:
Done